.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on: [ push, pull_request ]
+
+jobs:
+  test:
+    name: "JDK ${{ matrix.java }} on ${{ matrix.os }}"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+        java: [ 15 ] # [ 15, 14, ..., 11 ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: 'Check out repository'
+        uses: actions/checkout@v2
+      - name: 'Cache local Maven repository'
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: 'Set up JDK ${{ matrix.java }}'
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: 'Install'
+        shell: bash
+        run: mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+      - name: 'Test'
+        shell: bash
+        run: mvn test -B
+
+  publish_snapshot:
+    name: 'Publish snapshot'
+    needs: test
+    if: github.event_name == 'push' && github.repository == 'google/google-java-format' && github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check out repository'
+        uses: actions/checkout@v2
+      - name: 'Cache local Maven repository'
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: 'Set up JDK 15'
+        uses: actions/setup-java@v1
+        with:
+          java-version: 15
```

Comment:
Is there a `--release X` argument (or `--source/--target X`) in the Maven build? Otherwise, this will build GJF for Java SE 15.

Reply:
The maven build is setting `-source 1.8 -target 1.8`:

https://github.com/google/google-java-format/blob/3ce093515ad0bc1458e8965fdf71f78482f87968/pom.xml#L97
https://github.com/google/google-java-format/blob/3ce093515ad0bc1458e8965fdf71f78482f87968/pom.xml#L181

I meant to ask about the Java versions, though--we're currently testing with JDK 11 and 14, can we keep those version for now? Using 15 instead of 14 might be fine, but I want to test compatibility with 11.

Reply:
Sure, I'll add 11 back to the test matrix.